topology/m4/dai.m4

```diff
@@ -128,7 +128,7 @@ dnl DAI_CONFIG(type, idx, link_id, name, ssp_config/dmic_config)
 define(`DAI_CONFIG',
 `SectionHWConfig."'$1$2`" {'
 `'
-`	id		"'$2`"'
+`	id		"'$3`"'
```

Comment:
If you would like to make this field to be link_id, then you should align this value with the 

>  `	default_hw_conf_id	"'$2`"'

in line 148.

They should be same value, no matter how you define them.

Reply:
@xiulipan yes you are right. let me update it.